File: src/levels/L14-template-method/ts/base_manga.ts
class CoffeeRecipe {
  // region: CoffeeRecipe#make
  make() {
    this.boilWater()
    this.extract()
    this.pourInCup()
    this.putLid()
    this.addSleeve() // ← cambio 1
  }
  // endregion
}

class TeaRecipe {
  // Copia de la receta del café… a la que se le olvidó la tapa.
  // region: TeaRecipe#make
  make() {
    this.boilWater()
    this.steep()
    this.pourInCup()
    this.addSleeve() // ← cambio 2: la misma línea en la otra copia
  }
  // endregion
}

File: src/levels/L14-template-method/base_manga.rb
class CoffeeRecipe
  # region: CoffeeRecipe#make
  def make
    boil_water
    extract
    pour_in_cup
    put_lid
    add_sleeve # ← cambio 1
  end
  # endregion
end

class TeaRecipe
  # Copia de la receta del café… a la que se le olvidó la tapa.
  # region: TeaRecipe#make
  def make
    boil_water
    steep
    pour_in_cup
    add_sleeve # ← cambio 2: la misma línea en la otra copia
  end
  # endregion
end
